setup.py

```diff
@@ -37,6 +37,8 @@
         'requests',
         'koji',
         'PyGObject',
+        'reflink',
+        'pytest-runner',
```

Comment:
Now that it's in build dependencies, pytest-runner doesn't need to be here anymore

Reply:
ok will try and test it again

Reply:
ok it works